docker/main/docker-deploy.yml

```diff
@@ -10,7 +10,6 @@ services:
         - "3013:3013"
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
-      - ../../${DOCKER_VOL_DOMAINS?}:/nwm/domains
```

Comment:
@hellkite500, don't we need this to pull the domain data into the environment at runtime?

Reply:
This is mounted in to the individual workers and configured via the `load_image_and_mounts` in the `Launcher` class of `scheduler.py`